e2e/translates.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { test, expect } from './groups/fixture';
+import { initAsUsualUser } from 'e2e/helpers/e2e_auth';
+import { apiUrl } from 'e2e/helpers/e2e_http';
+
+test.use({ baseURL: 'http://localhost:4100' });
```

Comment:
You should specify that the "fr" translation has been run on this port.

Reply:
Fixed